### DISCUSSION TITLE
FEATURE REQUEST: Show the charging limit set in the car

### DISCUSSION THREAD

**Discussion by palengh** [pypolestar/polestar_api#122] [Ideas]
i would love to get the charginglimit set in the car, eg it´s set to 90% as recommended by Polestar. Some times before a long haul I set it to 100%, so would be nice if this was available through the integration :-)

**Comment by palengh** [comment]
We could do with just reading this value, making a sensor in HA

**Comment by palengh** [comment]
This would be related to #120; estimated charging time to full

We don't usually charge to 100%. Not me. Maybe max 80% on a long haul, and daily 90% at home.

**Comment by leeyuentuen** [comment]
it would be nice, but the issue is that we didn't have access to that information.

contact polestar without any luck, so I (or someone else) need to use them in other way to get this information

**Reply by storacm** [reply]
Setting the charge level i s now possible in the polestar app. Can you integrate this into the poletsar_api integrations?

**Reply by pabutterworth** [reply]
This would be very useful, is it still not available?

**Reply by jschlyter** [reply]
Not yet.

**Reply by pabutterworth** [reply]
ok thanks
